typing/typemod.ml

```diff
@@ -2020,6 +2182,11 @@ let report_error ppf = function
       fprintf ppf
         "This is an alias for module %a, which is missing"
         path p
+  | Invalid_open _me ->
+      fprintf ppf "Invalid open"
```

Comment:
This error message is a bit dry. Why is the open invalid? Why is the module expression not used?

Reply:
now `Invalid_open` is no longer used so I removed it.